docs/eventing/sources/apiserversource/getting-started/_index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Getting started"
+weight: 01
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - /docs/eventing/samples/kubernetes-event-source
+---
+
+## Prerequisites
+
+Before you can create an API server source, you must install Knative Eventing and the `kubectl` CLI tool.
```

Comment:
Note to self: we may want to add a link to Prerequisites page here once the split (#3399) is merged

Reply:
I think we can do that in a follow up PR at some point, maybe create an issue to standardize all the prerequisites for procedures?

Reply:
Abso-lutely :) I'll set up an issue after #3399 gets merged

Reply:
@omerbensaadon ping - reminder to create the issue since it looks like that's merged now ;)